server/src/services/user.service.ts

```diff
@@ -123,6 +125,47 @@ export class UserService {
     });
   }
 
+  getLicense({ user }: AuthDto): LicenseResponseDto {
```

Comment:
Where should we put in the mechanism that if the server license is set, we will return the valid status for each user license?